istioctl/cmd/proxyconfig.go

```diff
@@ -1412,6 +1412,10 @@ func getPodNames(podflag string) ([]string, string, error) {
 }
 
 func getPodName(podflag string) (string, string, error) {
+	return getPodNameWithDefaultNamespace(podflag, defaultNamespace)
+}
+
+func getPodNameWithDefaultNamespace(podflag, defaultNamespace string) (string, string, error) {
```

Comment:
I think the names here are misleading because it's not with default namespace.
Depends on the ns you provide which can be anything.
I suggest changing to `func getPodNameWithNamespace(podflag, namespace string)`.

Reply:
Thanks for the suggestion!